otx/cli/manager/config_manager.py

```diff
@@ -186,7 +186,7 @@ def _get_train_type(self, ignore_args: bool = False) -> str:
             if arg_algo_backend:
                 train_type = arg_algo_backend.get("train_type", {"value": "INCREMENTAL"})  # type: ignore
                 return train_type.get("value", "INCREMENTAL")
-            if self.mode in ("build") and self.args.train_type:
+            if self.mode in ("build", "train") and self.args.train_type:
```

Comment:
I think we should first check whether train_type exists in args.
False should be returned if train_type does not exist.